src/Stratis.Bitcoin.Features.PoA/BasePoAFeatureConsensusRules/PoAHeaderSignatureRule.cs

```diff
@@ -29,10 +41,17 @@ public override void Initialize()
             this.slotsManager = engine.SlotsManager;
             this.federationHistory = engine.FederationHistory;
             this.validator = engine.PoaHeaderValidator;
+
+            var lastCheckPoint = engine.Network.Checkpoints.LastOrDefault();
+            this.lastCheckPoint = (lastCheckPoint.Value != null) ? new HashHeightPair(lastCheckPoint.Value.Hash, lastCheckPoint.Key) : null;
         }
 
         public override async Task RunAsync(RuleContext context)
         {
+            // Only start validating at the last checkpoint block.
+            if (this.initialBlockDownloadState.IsInitialBlockDownload() && context.ValidationContext.ChainedHeaderToValidate.Height < (this.lastCheckPoint?.Height ?? 0))
```

Comment:
if chained header height < last checkpoint height then IBD is always true. So IsInitialBlockDownload check can be removed.